css/instruction.css

```diff
@@ -0,0 +1,35 @@
+@import url('./style.css');
+main .instruction-categories{
+    padding: 2em 5em;
+    flex-direction: column;
+    align-items: center;
+}
+main .instruction{
+    width: 100%;
+}
+.instruction ul li{
+    color: var(--gray-75);
+    margin: 1em 0;
+    font-size: 1.2rem;
+}
+.instruction ul li i{
+    font-size: 1rem;
+    color: var(--gray-80);
+    margin-right: 1em;
+}
+.instruction .confirm-checkbox{
+    background-color: var(--gray-00);
+    padding: 2em;
+    position: relative;
+}
+.instruction .btn-start-quiz{
+    text-align: center;
+    background-color: var(--gray-80);
+    border: none;
+    color: var(--gray-00);
+    margin-top: 0.5em;
+    margin-bottom: 1em;
+    position: absolute;
+    top: 0.2em;
+    right: 1em;
```

Comment:
any specific reason to use em?

Reply:
ye it will increase or decrease according to the current element's font size.